torch/_export/graph_module.py

```diff
@@ -291,15 +290,18 @@ def _get_submodule(
 
 
 def get_control_flow_submodules(
-    graph_module: ExportGraphModule,
-) -> List[Tuple[str, ExportGraphModule, torch.fx.Node]]:
+    graph_module: Union[ExportedProgram, torch.fx.GraphModule],
+) -> List[Tuple[str, torch.fx.GraphModule, torch.fx.Node]]:
     """
     Returns a list of submodules used for control flow operations
     (torch.ops.cond/map) that are in the given toplevel graph (does not look
     into submodules). Specifically, the returned value is a list containing a
     tuple of (name of the submodule that's stored in the graph module, the
     submodule itself, and the fx node that uses this submodule).
     """
+    if isinstance(graph_module, ExportedProgram):
+        graph_module = graph_module.graph_module
```

Comment:
Related to above, this looks kinda confusing. Do you think if there is a way to make this utility function only work for one type?